LowNotesApp/LowNotesApp/LowNotesApp.csproj

```diff
@@ -31,14 +31,11 @@
     <WarningLevel>4</WarningLevel>
```

Comment:
How our DB is created?

Reply:
I've created DB using SQL Managment Studio 2019.
Just simply made two tables with 3 rows in them and insert.
![image](https://user-images.githubusercontent.com/44684801/147344687-01f592cf-e47c-4eec-a8b7-788646d1693b.png)